incubator/logstash/README.md

```diff
@@ -2,6 +2,8 @@
 
 [Logstash](https://www.elastic.co/products/logstash) is an open source, server-side data processing pipeline that ingests data from a multitude of sources simultaneously, transforms it, and then sends it to your favorite “stash.”
 
+**Note - this chart has been deprecated and [moved to stable](../../stable/burrow)**.
```

Comment:
why burrow?

Reply:
I assume this is a Ctrl+C Ctrl+V gone wrong

Reply:
oops.
Stoopid cut and paste